### PR TITLE
PR-78 Skip jira-lint when all commits start with 'docs:'

### PR DESCRIPTION
**Pull request by andyasberry**
Please add anyone you feel would be interested.

* Successful run for all commits `docs:`, no key in branch name or PR title - https://github.com/invitation-homes/toolbelt-bff/actions/runs/4265041222/jobs/7423897802
* Behaves as it previously has if all commits do not start with `docs:` - https://github.com/invitation-homes/toolbelt-bff/actions/runs/4265054678/jobs/7423929408

Follow-up will be to (1) create a new tag and (2) move the `v2` tag.

### CODE REVIEW

**Review by ryan-breidenbach — APPROVED** [review]

File: src/main.ts [hunk 1]
```diff
@@ -121,6 +122,23 @@ async function run(): Promise<void> {
...
 
+    // skip if only `docs:` commits
+    // Get commits for pull request
+    const prPayload = {
+      owner,
+      repo,
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      pull_number: prNumber,
+    };
+    console.log('Fetching PR commits...');
```
Comment (edwagner) [inline src/main.ts:133]: nit: is this log message necessary, or a holdover from development? (ditto for the log after the commits are fetched)

**Review by edwagner — APPROVED** [review]
Thanks Andy!

File: src/main.ts (hunk 1 above)
Comment (andyasberry) [reply src/main.ts:133]: It's following the approach of the project we formed from (there are lots of these and I chose consistency, but it makes me cringe a little haha)

It should be using core.debug or something instead.

Comment (andyasberry) [reply src/main.ts:133]: Let me know if you think we should change them! Happy to hear your thoughts. Thanks!

Comment (edwagner) [reply src/main.ts:133]: Don't change it - consistency is a good reason.

**Pull request merged**